schema/spacewalk/common/views/suseChannelUserRoleView.sql

```diff
@@ -64,8 +66,14 @@ AS
         FROM rhnUserGroupMembers m
           JOIN rhnUserGroup g ON g.id = m.user_group_id
           JOIN rhnUserGroupType t ON t.id = g.group_type
-        WHERE t.label = 'channel_admin' OR t.label = 'org_admin') adminUsers
+        WHERE t.label = 'org_admin') adminUsers
      ON (adminUsers.user_id = combination.user_id)
+     LEFT JOIN
+       (SELECT DISTINCT uag.user_id, 1 AS is_channel_admin
+        FROM access.userAccessGroup uag
+          JOIN access.accessGroup ag ON ag.id = uag.group_id
+        WHERE ag.label = 'channel_admin') channelAdmins
```

Comment:
Just as a double check, we should also verify that the org for the access group is null.